icecap/ci/populate/default.nix

```diff
@@ -0,0 +1,70 @@
+let
+  icecapRev = builtins.getEnv "ICECAP_REV";
+  salt = builtins.getEnv "SALT";
+
+  icecapRemote = builtins.fetchGit rec {
+    url = "https://gitlab.com/arm-research/security/icecap/icecap-refs.git";
+    ref = "refs/tags/icecap/keep/${builtins.substring 0 32 rev}";
+    rev = icecapRev;
+    submodules = true;
+  };
+
+  icecapLocal = ../../../icecap;
+
+  icecapSource = icecapRemote;
+
+  # NOTE to develop using a local checkout of IceCap, replace the above with:
+  # icecapSource = icecapLocal;
+
+  icecap = import icecapSource;
+
+in with icecap;
+let
+
+  configured = pkgs.none.icecap.configured.virt;
+
+  inherit (configured) icecapFirmware icecapPlat mkDynDLSpec mkRealm;
+  inherit (pkgs.dev) writeText linkFarm;
+  inherit (pkgs.none.icecap) platUtils;
+  inherit (pkgs.linux.icecap) linuxKernel nixosLite;
+
+  run = platUtils.${icecapPlat}.bundle {
+    firmware = icecapFirmware.image;
+    payload = icecapFirmware.mkDefaultPayload {
+      linuxImage = linuxKernel.host.${icecapPlat}.kernel;
+      initramfs = hostUser.config.build.initramfs;
+      bootargs = [];
+    };
+  };
+
+  hostUser = nixosLite.eval {
+    modules = [
+      {
+        initramfs.extraContentCommands = ''
+          # ${salt}
+        '';
+      }
+    ];
+  };
+
+  spec = mkRealm {
+    script = ./realm.py;
+    config = {
+      inherit salt;
+      realm_id = 0;
+      num_cores = 1;
+    };
+  };
+
+  roots = [
+    run
+    spec
+    pkgs.dev.icecap.rustc
+    pkgs.dev.icecap.cargo
```

Comment:
These are unnecessary now that we don't use IceCap's rust toolchain. However, this is a minor note, because their presence doesn't really matter.